src/scene/materials/material.js

```diff
@@ -68,6 +68,13 @@ class Material {
      */
     name = 'Untitled';
 
+    /**
+     * A unique id the user can assign to the material.
```

Comment:
Maybe add some words on what this is for and why a user might want to set it?

Reply:
updated